lib/Dialect/TritonGPU/Transforms/Pipeliner/MatmulLoopPipeline.cpp

```diff
@@ -141,6 +193,12 @@ createAsyncCopy(scf::ForOp &forOp, tt::LoadOp loadOp, Value alloc,
     auto sharedLoad =
         builder.create<ttg::LocalLoadOp>(loc, loadOp.getType(), viewLoad);
     loadOp->replaceAllUsesWith(sharedLoad->getResults());
+
+    // Prefetch load if is not MMAV3 and is used by the dot.
+    if (loadToInfo[loadOp].usedByDot) {
+      schedule.insert(wait, numStages - 2, numStages * 2);
+      schedule.insert(viewLoad, numStages - 2, numStages * 2);
+    }
```

Comment:
I'm not sure I understand this part. Did we use to prefetch?

Reply:
The comment is misleading, sorry. Those are the two instructions that were used in "prefetch extracts". I have simply moved this logic from `createSchedule` to `createCoarseSchedule`. Does it make sense?